openjdk/excludes/ProblemList_openjdk17-openj9.txt

```diff
@@ -35,7 +35,6 @@ java/lang/ClassLoader/EndorsedDirs.java	https://github.com/eclipse-openj9/openj9
 java/lang/ClassLoader/ExtDirs.java	https://github.com/eclipse-openj9/openj9/issues/3055	generic-all
 java/lang/ClassLoader/LibraryPathProperty.java	https://github.com/adoptium/aqa-tests/issues/1297	generic-all
 java/lang/ClassLoader/RecursiveSystemLoader.java	https://github.com/eclipse-openj9/openj9/issues/3178	generic-all
-java/lang/ClassLoader/nativeLibrary/NativeLibraryTest.java	https://github.com/adoptium/temurin-build/issues/248	generic-all
```

Comment:
NativeLibraryTest is seen to have failed on OpenJ9. So, please put it back in ProblemList_openjdk17-openj9.txt. We an have it removed from ProblemList_openjdk17.txt.

Reply:
Suggestion for line 38 is to leave it in, but reference the new issue that describes the new failure
`java/lang/ClassLoader/nativeLibrary/NativeLibraryTest.java	https://github.com/eclipse-openj9/openj9/issues/14349	generic-all`